ops/db-connect.sh

```diff
@@ -1,30 +1,87 @@
-#!/bin/bash
-
-# Call this to connect to cloud Postgres for a specific Data Repo instance.
-# For example, to connect to the dev datarepo database as user mm, run:
-# $ DB=datarepo SUFFIX=mm ENVIRONMENT=dev ./db-connect.sh
-# To connect to a user instance's stairway database, run:
-# $ DB=stairway-mm SUFFIX=mm ENVIRONMENT=dev ./db-connect.sh
+#!/bin/sh
+#
+# This script is used to connect to the TDR Cloud SQL database in a GCP project.
+#
+# It uses the gcloud CLI to get the credentials for the cluster, kubectl to port
+# forward to the SQL proxy pod, and psql to connect to the database.
+#
+# You MUST have gcloud, kubectl, jq, and psql installed to run this script. In
+# addition, you MUST be connected to the VPN to access the database.
+#
+# Usage: ./db-connect.sh
+#
+# The following environment variables may be set to override the defaults:
 #
-# If the connection times out, check that you are on the Broad VPN and are connected to correct
-# Kubernetes cluster.
+#   PROJECT     the GCP project that contains the Cloud SQL database
 #
-# If you get an AlreadyExists error, it's possible the pod didn't shut down properly. You can
-# delete it using kubectl, where ZZ is the SUFFIX.
-# $ kubectl --namespace ZZ delete pod ZZ-psql
+#   NAMESPACE   the Kubernetes namespace for the SQL proxy pod
+#               these can be listed with `kubectl get namespaces`
+#
+#   SECRET      the secret path for the database password in GCP Secrets Manager
+#               the paths are listed at https://github.com/broadinstitute/vault-migration-tool/blob/main/mappings/tdr.yaml
+#
+#   PORT        the local port to forward to the SQL proxy pod
+#
+#   DATABASE    the name of the database to connect to
+#
+
+set -eu
+
+# Default values that may be overridden by environment variables
+PROJECT="${PROJECT:-broad-jade-dev}"
+NAMESPACE="${NAMESPACE:-dev}"
+SECRET="${SECRET:-helm-datarepodb}"
+PORT="${PORT:-5432}"
+DATABASE="${DATABASE:-stairway}"
```

Comment:
IMO, `datarepo` would be a more sensible default database, but as a caller I'd like to know that `datarepo` or `stairway` are the two supported options.

Is there an opportunity to add a `--help` option to this script that prints out usage info?

Reply:
Yes, great idea :)